src/disputables/mappings.js

```diff
@@ -35,7 +35,18 @@ export const DISPUTABLE_ACTIONS = new Map([
 ])
 
 // Mapping of all disputable apps appId to their
-// corresponding subgraph name
-export const DISPUTABLE_SUBGRAPH_NAMES = new Map([
-  ...VOTING_APP_IDS.map(appId => [appId, 'aragon-dvoting']),
+// corresponding subgraph urls by network
+export const DISPUTABLE_SUBGRAPH_URLS = new Map([
+  ...VOTING_APP_IDS.map(appId => [
+    appId,
+    {
+      // TODO: Update to Aragon's own node
+      mainnet:
+        'https://api.thegraph.com/subgraphs/name/aragon/aragon-dvoting-mainnet-staging',
```

Comment:
Will update when we get green light.

Reply:
Let's update it to our own hosted version now—the network dashboard is already relying on it.